packages/templates/clients/websocket/python/components/QueryParamsArgumentsDocs.js

```
@@ -0,0 +1,19 @@
+export function QueryParamsArgumentsDocs({ queryParams }) {
+  if (!queryParams) {
+    return null;
+  }
+
+  return queryParams.map((param) => {
+    const paramName = param[0];
+    const firstLine = `${paramName} (str, optional):`;
+    const secondLine = `If provided (or if ${paramName.toUpperCase()} environment variable is set), added as ?${paramName}=… to URL`;
+    return (
+      <Text indent={1}>
+        {firstLine}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)